src/test/java/it/pn/frontend/e2e/stepDefinitions/common/BackgroundTest.java

```diff
@@ -64,6 +64,20 @@ public void loginPF(String nomeFileLogin) {
         deleghePagoPATest.visualizzaDelegheSection();
     }
 
+    public void loginPGDeleghe(String nomeFileLogin) {
+        loginPGPagoPATest.loginPortalePersonaGiuridicaTramiteTokenExchange(nomeFileLogin);
+        notifichePGPagoPATest.nellaPaginaPiattaformaNotifichePersonaGiuridicaClickSulBottoneDeleghe();
+        notifichePGPagoPATest.visualizzaDelegheSection();
+    }
+        public void accettazioneDelegaPG(){
+        deleghePagoPATest.siSceglieOpzioneAccetta();
+        deleghePagoPATest.siInserisceIlCodiceDelegaNelPopUp(nomeFileNuovaDelegaPG);
+        deleghePagoPATest.siCliccaSulBottoneAccetta();
+        deleghePGPagoPATest.siCliccaSulBottoneConfermaGruppo();
+
+        }
+
+
```

Comment:
There should already be an API background method available for what you're trying to accomplish

Reply:
Available ones weren't working for this case

Reply:
You're correct, there isn't any implementation related to the acceptance of the delegate. For now, this should be acceptable, especially considering that PF has the same adaptation. In future test scenarios, it would be useful to integrate background APIs.